bin/dfx-sns-sale-buy

```diff
@@ -42,4 +60,4 @@ echo "${@}"
   echo "Ticket creation time:"
   echo "$TICKET_CREATION_TIME"
   exit 1
```

Comment:
I try generally to send error messages to stderr; that is what stderr is there for.  But don't always remember to.  `[["$FAILED" == "false" ] || {...} >&2` would do the trick.  Or  however else you like.

Reply:
Done